src/accelerate/accelerator.py

```diff
@@ -2507,7 +2508,7 @@ def save_model(
                             del state_dict[name]
                             warn_names.add(name)
             if len(warn_names) > 0:
-                logger.warning_once(
+                logger.warning(
```

Comment:
Why was this changed?

Reply:
it was copy pasted from transformers but logger in accelerate don't have the `warning_once` function